File: src/main/java/cz/agents/amodsim/config/Stations.java
package cz.agents.amodsim.config;

import java.lang.Integer;
import java.lang.String;

public final class Stations {
  public final String demandFilePath;

  public final String distanceMatrixPath;

  public final Integer regions;

  public final Integer timestamps;

  public final String distanceCalculatorPath;

  public final String dir;

  public final String centroidsFilePath;

  public final String stationsFilePath;

  public final String distanceMatrixOutputPath;

  public final String smoothedDemandFilePath;

  public Stations(String demandFilePath, String distanceMatrixPath, Integer regions,
      Integer timestamps, String distanceCalculatorPath, String dir, String centroidsFilePath,
      String stationsFilePath, String distanceMatrixOutputPath, String smoothedDemandFilePath) {
    this.demandFilePath = demandFilePath;
    this.distanceMatrixPath = distanceMatrixPath;
    this.regions = regions;
    this.timestamps = timestamps;
    this.distanceCalculatorPath = distanceCalculatorPath;
    this.dir = dir;
    this.centroidsFilePath = centroidsFilePath;
    this.stationsFilePath = stationsFilePath;
    this.distanceMatrixOutputPath = distanceMatrixOutputPath;
    this.smoothedDemandFilePath = smoothedDemandFilePath;
  }
}

File: src/main/python/amodsim/config/stations.py

class Stations:
    def __init__(self, properties: dict=None):
        self.regions = properties.get("regions")
        self.timestamps = properties.get("timestamps")
        self.dir = properties.get("dir")
        self.stations_file_path = properties.get("stations_file_path")
        self.demand_file_path = properties.get("demand_file_path")
        self.centroids_file_path = properties.get("centroids_file_path")
        self.distance_calculator_path = properties.get("distance_calculator_path")
        self.distance_matrix_output_path = properties.get("distance_matrix_output_path")
        self.distance_matrix_path = properties.get("distance_matrix_path")
        self.smoothed_demand_file_path = properties.get("smoothed_demand_file_path")


        pass

